docs/topics/preview_support.md

```diff
@@ -44,6 +45,31 @@ roborazzi {
 }
 ```
 
+<details>
+<summary>build.gradle version</summary>
+<br>
+
+```groovy
+roborazzi {
+    generateComposePreviewRobolectricTests.enable.set(true)
+    // The package names to scan for Composable Previews.
+    generateComposePreviewRobolectricTests.packages.set(["com.example"])
+    // robolectricConfig will be passed to Robolectric's @Config annotation in the generated test class.
+    // See https://robolectric.org/configuring/ for more information.
+    generateComposePreviewRobolectricTests.robolectricConfig.set([
+        "sdk": "[32]",
+        "qualifiers": "RobolectricDeviceQualifiers.Pixel5",
+    ])
+    // If true, the private previews will be included in the test.
+    generateComposePreviewRobolectricTests.includePrivatePreviews.set(true)
+    // The fully qualified class name of the custom test class that implements [com.github.takahirom.roborazzi.ComposePreviewTester].
+    generateComposePreviewRobolectricTests.testerQualifiedClassName.set("com.example.MyCustomComposePreviewTester")
+}
+```
```

Comment:
```groovy
// In build.gradle
roborazzi {
    generateComposePreviewRobolectricTests {
        enable = true
        packages = ....
        .....
    }
}
```

Error message
```
Caused by: groovy.lang.MissingPropertyException: Could not set unknown property 'enable' for extension 'roborazzi' of type io.github.takahirom.roborazzi.RoborazziExtension.
```

This way of writing will cause the following error and gradle sync will not succeed